src/indexed_crate.rs

```diff
@@ -211,6 +211,9 @@ pub struct IndexedCrate<'a> {
     /// index: kind of public top-level item -> `IndexMap<Id, &'a Item>` of that kind
     pub(crate) pub_item_kind_index: PubItemKindIndex<'a>,
 
+    /// index: enum id + variant name -> Variant
+    pub(crate) variant_name_index: Option<HashMap<(Id, &'a str), (&'a Item, usize)>>,
```

Comment:
Make sure to document the value precisely. Presumably the `usize` is the index of the variant inside its enum definition?